src/deployment/bicep-templates/autoscale-settings.bicep

```diff
@@ -28,7 +30,7 @@ resource autoscaleSettings 'Microsoft.Insights/autoscalesettings@2015-04-01' = {
               metricResourceUri: server_farm_id
               operator: 'GreaterThanOrEqual'
               statistic: 'Average'
-              threshold: 20
+              threshold: 50
               timeAggregation: 'Average'
               timeGrain: 'PT1M'
               timeWindow: 'PT1M'
```

Comment:
Should this be changed to 'PT10M' to mirror prod?

Reply:
You're right I needed to update time, but it was 'cooldown' that I needed to update.